ingestion_tools/dataset_configs/10427.yaml

```diff
@@ -12,65 +12,237 @@ datasets:
       corresponding_author_status: true
-      ORCID: 0000-0002-5130-5061
-    - name: Justin W. Taraska
+      name: Justin W. Taraska
       primary_author_status: false
-      corresponding_author_status: true
-      ORCID: 0000-0001-5355-9535
-    funding:
-      - funding_agency_name: Chan Zuckerberg Initiative
-        grant_id: 2021-234544
-      - funding_agency_name: NIH
-        grant_id: 1ZIAHL006098
-      - funding_agency_name: NIH
-        grant_id: 1ZIADK060100
     cell_type:
-      name: RCB1975 cell
       id: CLO:0051569
+      name: RCB1975 cell
     cross_references:
       dataset_publications: 10.1101/2024.06.27.600657
-    dataset_description: "Michalak, Sun, Sochacki et al. 2024 unroofing manuscript contains 10 grids. This is grid 2 of
-      10 and contains basal membranes of HSC3 cells.  The cell line expresses EGFR-GFP as described in Panilla-Macua et
-      al. Elife 2017 and has been serum starved.  The cells were grown on Quantifoil Au 300 R2/2, 2 nm carbon grids
-      prior to removing the apical membrane and cytoplasm with 0.7 bar of paraformaldehyde prior to vitrification."
+    dataset_description: Michalak, Sun, Sochacki et al. 2024 unroofing manuscript
+      contains 10 grids. This is grid 2 of 10 and contains basal membranes of HSC3
+      cells.  The cell line expresses EGFR-GFP as described in Panilla-Macua et al.
+      Elife 2017 and has been serum starved.  The cells were grown on Quantifoil Au
+      300 R2/2, 2 nm carbon grids prior to removing the apical membrane and cytoplasm
+      with 0.7 bar of paraformaldehyde prior to vitrification.
     dataset_identifier: 10427
     dataset_title: Unroofing Grid02
-    dates:
+    dates: &id002
       deposition_date: '2024-07-01'
       last_modified_date: '2024-07-01'
       release_date: '2024-07-05'
-    grid_preparation: "plasma discharge 30 sec + UV treatment 20 min + 125 ng/ul fibronectin, 20 minutes + water rinse +
-      seeded with 15000 cells"
+    funding:
+    - funding_agency_name: Chan Zuckerberg Initiative
+      grant_id: 2021-234544
+    - funding_agency_name: NIH
+      grant_id: 1ZIAHL006098
+    - funding_agency_name: NIH
+      grant_id: 1ZIADK060100
+    grid_preparation: plasma discharge 30 sec + UV treatment 20 min + 125 ng/ul fibronectin,
+      20 minutes + water rinse + seeded with 15000 cells
     organism:
       name: Homo sapiens
       taxonomy_id: 9606
-    sample_preparation: serum starved and grown on Quantifoil Au 300 R2/2, 2 nm carbon grids
+    sample_preparation: serum starved and grown on Quantifoil Au 300 R2/2, 2 nm carbon
+      grids
     sample_type: cell
   sources:
   - literal:
       value:
       - '10427'
+depositions:
+- metadata:
+    authors: *id001
+    dates: *id002
+    deposition_description: TBA
+    deposition_identifier: 10306
+    deposition_title: TBA
+    deposition_type:
+    - dataset
+  sources:
+  - literal:
+      value:
+      - 10306
+frames:
+- sources:
+  - parent_filters:
```

Comment:
I think most of these sources have a lot in common... does the converter script need to be smart enough to just smash them all into a single source?